docs/source/tutorial/create_pipelines.md

```diff
@@ -540,8 +540,8 @@ from kedro_tutorial.pipelines import data_science as ds
 * The two modular pipelines are merged together into a project `__default__` pipeline using the `+` operator.
 * The `data_processing_pipeline` will preprocess the data, and `data_science_pipeline` will create features, train and evaluate the model.
 
-```eval_rst
-.. note::  The order in which you add the pipelines together is not significant (``data_science_pipeline + data_processing_pipeline`` would produce the same result), since Kedro automatically detects the data-centric execution order for all the nodes in the resulting pipeline.
+```{note}
+The order in which you add the pipelines together is not significant (``data_science_pipeline + data_processing_pipeline`` would produce the same result), since Kedro automatically detects the data-centric execution order for all the nodes in the resulting pipeline.
```

Comment:
```suggestion
The order in which you add the pipelines together is not significant (`data_science_pipeline + data_processing_pipeline` would produce the same result), since Kedro automatically detects the data-centric execution order for all the nodes in the resulting pipeline.
```